ldap_jwt_auth/routers/verify.py

```diff
@@ -0,0 +1,33 @@
+"""
+Module for providing an API router which defines a route for managing verification of JWT tokens.
+"""
+import logging
+from typing import Annotated
+
+from fastapi import APIRouter, Body, Depends, HTTPException, Response, status
+
+from ldap_jwt_auth.auth.jwt_handler import JWTHandler
+from ldap_jwt_auth.core.exceptions import InvalidJWTError
+
+logger = logging.getLogger()
+
+router = APIRouter(prefix="/verify", tags=["authentication"])
+
+
+@router.post(
+    path="/",
+    summary="Verify that a JWT token was generated by this authentication service",
+    response_description="200 status code (no response body) if the token is valid",
+)
+def verify_token(
+    jwt_handler: Annotated[JWTHandler, Depends(JWTHandler)],
+    token: Annotated[str, Body(description="The JWT token to verify")],
```

Comment:
This actually doesnt work, I found https://github.com/ral-facilities/operationsgateway-api/blob/e2d9b4ccf0f79f5ece3404a2eeb8178b4b7dfb74/operationsgateway_api/src/routes/auth.py#L18-L24 confusing, but it actually needs a body like
`
{
token: token_string
}
`
instead of 
`
{
token_string
}
`
I think they must have used Body like that so it could be put anywhere and be reused, but here it would be fine to just have a VerifySchema or something.

I also realise the login one is in `models.py`, should that be `schemas.py`?

Reply:
First point addressed in https://github.com/ral-facilities/ldap-jwt-auth/pull/9/commits/54fde2dd237f7a1c321490367aee47e3ee04e5fd and second in https://github.com/ral-facilities/ldap-jwt-auth/pull/9/commits/3a908dc03e0052cf46f6070f3deaa31d9642744a